cas-server-webapp/src/main/webapp/WEB-INF/webflow/login/login-webflow.xml

```diff
@@ -110,7 +110,8 @@
         <transition on="FailedLoginException" to="generateLoginTicket"/>
         <transition on="AccountNotFoundException" to="generateLoginTicket"/>
         <transition on="UnauthorizedServiceForPrincipalException" to="generateLoginTicket" />
-        <transition on="UNKNOWN" to="generateLoginTicket"/>
```

Comment:
Based on your impl above, it's possible for UNKNOWN to be returned. So this likely should not be removed.